### PR TITLE
Update appendices.tex

### PR DESCRIPTION
**Pull request by quaere-verum**
Corrected erroneous statements, streamlined the section in integrability of geometric structures.
- Bram

### CODE REVIEW

File: appendices.tex [hunk 1]
```diff
@@ -90,105 +90,91 @@ \section{Prelude to Lie Algebroids}
...
-          %https://math.mit.edu/classes/18.157/S2012/lecturesonsymplecticgeometryanacannas.pdf
-  \end{enumerate}
-\end{proof}
 \begin{corollary}
-  Any complex manifold with a metric has a symplectic structure defined by $\omega=g\circ (J\otimes\text{id})$. In particular, any projective complex manifold $Z\subset\mathbb{CP}^n$ has a canonical symplectic structure defined by $\omega(X,Y)=\iota^*h(J(X),Y)$.
+    The $2$-form defined by $\omega=h\circ J\otimes\text{id}$ is closed and non-degenerate. That is, $(\mathbb{CP}^n, \omega)$ is a symplectic manifold.
 \end{corollary}
+Notice that we are dealing with a very special manifold here. It has \textit{two} integrable structures. A complex structure, and a symplectic structure, which are compatible in the sense that $\omega(\cdot, J\cdot)$ defines a metric.
+\begin{definition}
+    A complex symplectic manifold $(M,\omega, J)$ such that $g=\omega(\cdot, J\cdot)$ defines a metric is called a Kähler manifold.
```
Comment (mseri) [inline appendices.tex]:
```suggestion
    A complex symplectic manifold $(M,\omega, J)$ such that $g=\omega(\cdot, J\cdot)$ defines a metric is called a \emph{Kähler manifold}.
```

File: appendices.tex [hunk 2]
```diff
@@ -90,105 +90,91 @@ \section{Prelude to Lie Algebroids}
...
-%https://arxiv.org/abs/alg-geom/9411018
-The homological mirror symmetry conjecture (which has been verified for a large number of examples) uses the symplectic structure on a manifold to construct the so-called Fukaya category. The conjecture asserts that this category is equivalent to a certain category defined over its mirror pair.\\
-However, we cannot even hope to touch upon mirror symmetry here. Instead, we will carry on to prove our assertion that complex projective space has a canonical symplectic structure. The main result that we need is the following.
+There is another prototypical symplectic manifold.
 \begin{theorem}
-  Complex projective space $\mathbb{CP}^n$ inherits a metric $h$ from $\mathbb{C}^{n+1}$ called the Fubini-Study metric.
+  Complex projective space $\mathbb{CP}^n$ inherits a metric $h$ from $\mathbb{C}^{n+1}$ called the Fubini-Study metric. Additionally, with respect to the Levi-Civita connection of $h$, it holds that $\nabla J=0$.
```
Comment (mseri) [inline appendices.tex]:
```suggestion
  Complex projective space $\mathbb{CP}^n$ inherits a metric $h$ from $\mathbb{C}^{n+1}$ called the \emph{Fubini-Study metric}. Additionally, with respect to the Levi-Civita connection of $h$, it holds that $\nabla J=0$.  
```

File: appendices.tex [hunk 3]
```diff
@@ -90,105 +90,91 @@ \section{Prelude to Lie Algebroids}
...
+\end{example}
 Next, we have an example from the realm of Poisson geometry. This is in some sense a generalisation of symplectic geometry. The theory of Poisson manifolds is a very active research area, related to quantisation of physical theories, as well as modelling constraints on physical systems.
 \begin{definition}
-  An almost Poisson manifold is a pair $(M,\Pi)$ such that the bracket $\{\cdot,\cdot\}:C^\infty(M)\times C^\infty(M)\to C^\infty(M)$ defined by $\{f,g\}=\Pi(df,dg)$ satisfies the axioms of a Lie algebra, save for the Jacobi identity. If the bracket satisfies the Jacobi identity, the pair is called a Poisson manifold.
+  An almost Poisson manifold is a pair $(M,\pi)$ such that the bracket $\{\cdot,\cdot\}:C^\infty(M)\times C^\infty(M)\to C^\infty(M)$ defined by $\{f,g\}=\pi(df,dg)$ satisfies the axioms of a Lie algebra, save for the Jacobi identity. If the bracket satisfies the Jacobi identity, the pair is called a Poisson manifold.
```
Comment (mseri) [inline appendices.tex]:
```suggestion
  An \emph{almost Poisson manifold} is a pair $(M,\pi)$ such that the bracket $\{\cdot,\cdot\}:C^\infty(M)\times C^\infty(M)\to C^\infty(M)$ defined by $\{f,g\}=\pi(df,dg)$ satisfies the axioms of a Lie algebra, save for the Jacobi identity. If the bracket satisfies the Jacobi identity, the pair is called a \emph{Poisson manifold}.
```

Comment (mseri) [reply appendices.tex]: I liked $\Pi$ more, we used $\pi$ usually for projections

Comment (mseri) [reply appendices.tex]: I'd like to cite the recent book by Krainic and Marcut here, for readers that would like to know more. Here is the DOI:
```
@book{Crainic_2021,
  title={Lectures on Poisson Geometry},
  ISBN={9781470466671},
  ISSN={1065-7339},
  url={http://dx.doi.org/10.1090/gsm/217},
  DOI={10.1090/gsm/217},
  journal={Graduate Studies in Mathematics},
  publisher={American Mathematical Society},
  author={Crainic, Marius and Fernandes, Rui and Mărcuţ, Ioan},
  year={2021}
}
```

File: appendices.tex [hunk 4]
```diff
@@ -90,105 +90,91 @@ \section{Prelude to Lie Algebroids}
...
+    Let $(M,\pi)$ be an almost Poisson manifold. Define
+    \begin{equation}
+        L=\{\iota_\eta \pi +\eta\mid\eta\in T^*M\}\subseteq \mathbb{T}M
+    \end{equation}
+    Then $L$ is a Dirac structure if and only if $(M,\pi)$ is a Poisson manifold.
+\end{example}
+
 \begin{definition}
-  A Lie algebroid is a vector bundle $E\xrightarrow{\pi}M$ with a bracket $[\cdot,\cdot]$ such that $(\Gamma(E),[\cdot,\cdot])$ is a Lie algebra. Furthermore, there is a vector bundle morphism $\rho:E\to TM$ called the anchor, such that $[X,f\cdot Y]=\rho(X)f\cdot Y+f\cdot[X,Y]$ for all $X,Y\in\Gamma(E)$, $f\in C^\infty(M)$.
+  A Lie algebroid is a vector bundle $E\xrightarrow{\pi}M$ together with
```
Comment (mseri) [inline appendices.tex]:
```suggestion
  A \emph{Lie algebroid} is a vector bundle $E\xrightarrow{\pi}M$ together with
```

File: appendices.tex [hunk 5]
```diff
@@ -208,17 +194,25 @@ \section{Generalising the Exterior Derivative}
   \end{enumerate}
 \end{proposition}
 Notice how these properties are analogous to those of the exterior derivative. If $f,g\in C^\infty(M)$ and $X\in\Gamma(TM)$, then $df(gX)=g\cdot df(X)$ and $d(f\cdot g)(X)=df(X)\cdot g+f\cdot df(X)$.
-\begin{definition}
+\begin{definition}\label{def:linearConnection}
   Let $E\xrightarrow{\pi}M$ be an arbitrary vector bundle. Any $\mathbb{R}$-linear map $\nabla:\Gamma(TM)\times\Gamma(E)\to\Gamma(E)$ satisfying the above two properties is called a connection on $E$. That is:
```
Comment (mseri) [inline appendices.tex]:
```suggestion
  Let $E\xrightarrow{\pi}M$ be an arbitrary vector bundle. Any $\mathbb{R}$-linear map $\nabla:\Gamma(TM)\times\Gamma(E)\to\Gamma(E)$ satisfying the above two properties is called a \emph{connection} on $E$. That is:
```

**Comment by mseri** [comment]
Thanks for the update. Would it be possible to update it with the changes above? Otherwise I'll do it in the next few days and merge

**Comment by quaere-verum** [comment]
Incorporated the suggested changes.

**Review by mseri — APPROVED** [review]

**Comment by mseri** [comment]
Thanks a lot

**Pull request merged**